app/service/send_notification.py

```diff
@@ -105,7 +105,8 @@ def get_reply_to_text(
         try:
             if notification_type == EMAIL_TYPE:
                 message = 'Reply to email address not found'
-                reply_to = dao_get_reply_to_by_id(service.id, sender_id).email_address
+                reply_to = ''
+                current_app.logger.warning("Couldn't find the reply-to e-mail address.")
```

Comment:
This previously raised a `BadRequestError`. We'd be better off just raising the exception if this always did.